redash/security.py

```diff
@@ -0,0 +1,28 @@
+from flask_talisman import talisman
+
+from redash import settings
+
+talisman = talisman.Talisman()
+
+
+def init_app(app):
+    talisman.init_app(
+        app,
+        feature_policy=settings.FEATURE_POLICY,
+        force_https=settings.ENFORCE_HTTPS,
+        force_https_permanent=settings.ENFORCE_HTTPS_PERMANENT,
+        force_file_save=settings.ENFORCE_FILE_SAVE,
+        frame_options=settings.FRAME_OPTIONS,
+        frame_options_allow_from=settings.FRAME_OPTIONS_ALLOW_FROM,
+        strict_transport_security=settings.HSTS_ENABLED,
+        strict_transport_security_preload=settings.HSTS_PRELOAD,
+        strict_transport_security_max_age=settings.HSTS_MAX_AGE,
+        strict_transport_security_include_subdomains=settings.HSTS_INCLUDE_SUBDOMAINS,
+        content_security_policy=settings.CONTENT_SECURITY_POLICY,
+        content_security_policy_report_uri=settings.CONTENT_SECURITY_POLICY_REPORT_URI,
+        content_security_policy_report_only=settings.CONTENT_SECURITY_POLICY_REPORT_ONLY,
+        content_security_policy_nonce_in=settings.CONTENT_SECURITY_POLICY_NONCE_IN,
+        referrer_policy=settings.REFERRER_POLICY,
+        session_cookie_secure=settings.SESSION_COOKIE_SECURE,
+        session_cookie_http_only=settings.SESSION_COOKIE_HTTPONLY,
```

Comment:
This plays along with Flask-Login?

Reply:
AFAIU since Flask-Login simply relies on the Flask session system for tracking login state and has [for a while](https://github.com/maxcountryman/flask-login/commit/312d0e4554d3e0dfb7cd3973d79a7f342d645bff) shipped with an own httpOnly functionality for its rememberme cookie.